### ISSUE TITLE
tests are failing

### ISSUE THREAD

**Issue by fmv1992** [DEAP/deap#169]
On python3 I get:
```
dist-packages/deap » pwd 
/usr/local/lib/python3.4/dist-packages/deap
dist-packages/deap » python3 -m unittest discover

                           Ommited for brevity [...]

....x...E
======================================================================
ERROR: test_pickle_tree (tests.test_pickle.Pickling)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/deap/tests/test_pickle.py", line 60, in test_pickle_tree
    msg =  "Unpickled individual %s != pickled individual %s" % (str(ind), str(ind_l))
  File "/usr/local/lib/python3.4/dist-packages/deap/gp.py", line 91, in __str__
    while len(stack[-1][1]) == stack[-1][0].arity:
AttributeError: 'builtin_function_or_method' object has no attribute 'arity'

----------------------------------------------------------------------
Ran 12 tests in 0.005s

FAILED (errors=1, expected failures=1)
```
On python2.7 I get the same message but on `line 90` instead of `line 91`.

I tried to debug the code but I do not know anything about PrimitiveTree.

**Comment by fmder** [comment]
What version of DEAP do you use? test_pickle_tree is in the 1.0 branch I think you should consider using the 1.1 version which is stable.

```pip install git+https://github.com/DEAP/deap.git@master```

**Comment by fmv1992** [comment]
That is correct. Thanks.

On the other hand you should consider taking PEP 453 into account which says:

> This PEP proposes that the Installing Python Modules guide in Python 2.7, 3.3 and 3.4 be updated to officially recommend the use of pip as the default installer for Python packages

When I do `pip3 install deap` I get version 1.0.

Great work with the software :)

**Comment by fmder** [comment]
That is because we did not released officially 1.1 yet.
